<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="getPushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
background-color: rgb(255, 132, 139); 
border-radius: 3px; color: rgb(255, 255, 255); 
} 
QPushButton:hover { background-color: rgb(32, 178, 170); }</string>
    </property>
    <property name="text">
     <string>获取USB设备</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>USB设备</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>100</y>
       <width>189</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>序列号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serLineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>60</y>
       <width>188</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>PID:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="pidLineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>20</y>
       <width>188</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>VID:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="vidLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>20</width>
      <height>471</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="sendPushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
background-color: rgb(255, 132, 139); 
border-radius: 3px; color: rgb(255, 255, 255); 
} 
QPushButton:hover { background-color: rgb(32, 178, 170); }</string>
    </property>
    <property name="text">
     <string>发送键值询问码0x0A</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="clearrecPushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除询问码</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>311</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>键值询问码</string>
    </property>
    <widget class="QPushButton" name="k5PushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k6PushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k7PushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k8PushButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k9PushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k10PushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k11PushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K11</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k12PushButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K12</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k13PushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K13</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k14PushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K14</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k15PushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K15</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k16PushButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K16</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="k9LineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k10LineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k11LineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k12LineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k13LineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k14LineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k15LineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k16LineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="k1PushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k4PushButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>310</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k2PushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="k3PushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>310</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>K3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="k1LineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k2LineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>350</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k3LineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>350</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k4LineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>350</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k5LineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k6LineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>260</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k7LineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="k8LineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>20</width>
      <height>451</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="closeAllPushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
background-color: rgb(205, 92, 92); 
border-radius: 3px; color: rgb(255, 255, 255); 
} 
QPushButton:hover { background-color: rgb(255, 0, 0); }</string>
    </property>
    <property name="text">
     <string>关闭整个页面</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>当前状态键键值：</string>
    </property>
    <widget class="QTextEdit" name="keyTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>221</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="queryPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
background-color: rgb(255, 132, 139); 
border-radius: 3px; color: rgb(255, 255, 255); 
} 
QPushButton:hover { background-color: rgb(32, 178, 170); }</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="queryclearPushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
